templates/react/utils/helpers.js

```diff
@@ -7,26 +7,17 @@ export function itemToLinks(items) {
 }
 
 function createLink(item) {
-  if (typeof(item) === 'string' && item.includes(API_PATH) > 0) {
-    let route = null;
-    const routeTest = item.replace(API_PATH, '');
-    if( routeTest[0] === '/') {
-      route = item.replace(API_PATH, '').split('/')[1];
-    } else {
-      route = item.replace(API_PATH, '').split('/')[0];
-    }
-    return (
-      <span key={item}>
-        <Link to={`/${route}/show/${encodeURIComponent(item)}`}>
-          {item}
-        </Link>
-        <br/>
-      </span>
-    );
+  if ('string' !== typeof(item) || !item.includes(API_PATH)) {
+    return <span key={item}>{item}<br/></span>;
```

Comment:
<br> really needed? Could `<p>` or `<div>` do the trick?

Reply:
Don't know, it was already like this. I'll try that.